lib/electric_slide/agent_strategy/fixed_priority.rb

```diff
@@ -30,8 +30,7 @@ def available_agent_summary
       end
 
       def checkout_agent
-        priorities = @priorities.sort.to_h
-        _, agents = priorities.detect do |priority, agents|
+        _, agents = @priorities.sort.detect do |priority, agents|
```

Comment:
Ops, I just noticed that you've made a comment on the other PR. Should I revert this?